controllers/configmap/configmap_controller.go

```diff
@@ -56,6 +56,18 @@ func (r *ConfigMapReconciler) initAVI(ctx context.Context,
 			log.Error(err, "Cannot init AVI clients from secrets")
 			return res, err
 		}
+		version, err := r.aviClient.GetControllerVersion()
```

Comment:
This is the bug fix right, reinitialize the client

Reply:
Yep